saleor/graphql/webhook/subscription_payload.py

```diff
@@ -54,10 +56,13 @@ def initialize_context() -> HttpRequest:
     return: HttpRequest
     """
     handler = BaseHandler()
-    context = RequestFactory().request()
+    context = RequestFactory().request(
+        **{"SERVER_NAME": SimpleLazyObject(lambda: settings.ALLOWED_HOSTS[0])}
```

Comment:
Just to confirm. Are you sure, that this is OK to provide only the first value of allowed hosts?
Why do we need a SimpleLazyObject? Is it requirement for request?

Reply:
Also why create a dict to immediately unpack it to keyword args?

Reply:
I think that in this case there is no matter which element of `ALLOWED_HOSTS` we choose, @maarcingebala suggested in the same way as in[ WSGI app](https://github.com/saleor/saleor/blob/ebc0e0e66168c358373d594c0c50865ef8e6507f/saleor/wsgi/__init__.py#L37) so I did that.

Reply:
@patrys Didn't we want to stop relying on domains from site settings and deprecate/drop the API to set it?

Reply:
OK so let's use site_settings then.

Reply:
Are domains from Site objects always added to ALLOWED_HOSTS?

Reply:
They better be or Saleor won't work 😄